backend/api/auth.py

```diff
@@ -0,0 +1,83 @@
+from datetime import timedelta
+
+from fastapi import APIRouter, Depends, HTTPException, status
+from fastapi.encoders import jsonable_encoder
+from fastapi.security import OAuth2PasswordBearer, OAuth2PasswordRequestForm
+
+from backend.core.auth import Auth
+from backend.core.config import settings
+from backend.database import database
+from backend.models.token_model import Token
+from backend.models.user_model import User, UserCreate
+
+
+auth = Auth()
+router = APIRouter()
+users_collection = database.users
+
+
+oauth2_scheme = OAuth2PasswordBearer(tokenUrl='token')
+
+
+@router.post('/token', response_model=Token)
+async def login_for_access_token(request: OAuth2PasswordRequestForm = Depends()):
+    user = users_collection.find_one({'username': request.username})
+    if not user:
+        raise HTTPException(
```

Comment:
Let's consider name somehow that exceptions and move them to the separated module, so it'd more readable